packages/translations/src/i18next/i18next.ts

```diff
@@ -40,7 +40,6 @@ export const getAllowedLanguages = () => {
         DE: 'Deutsch',
         KM: 'ខ្មែរ',
         KO: '한국어',
-        MN: 'Монгол',
```

Comment:
we added it to `UNSUPPORTED_LANGUAGES` is it required to remove the country code from allowed_languages 🤔

Reply:
yep. other packages like appstore, cfd are still using this